### PR TITLE
chore: add datagrid stories

### PR DESCRIPTION
**Pull request by steve-haar**
add datagrid stories

### CODE REVIEW

File: .storybook/stories/datagrid/datagrid.stories.ts [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import { action } from '@storybook/addon-actions';
+import { Parameters } from '@storybook/addons';
+import { Story } from '@storybook/angular';
+
+import { elements } from '../../helpers/elements.data';
+import { setupStorybook } from '../../helpers/setup-storybook.helpers';
+
+const defaultStory: Story = args => ({
+  template: `
+  <style>.electronegativity-container { border-bottom: 4px solid #119cd4; }</style>
```
Comment (kevinbuhmann) [inline .storybook/stories/datagrid/datagrid.stories.ts:17]: nitpick: This template isn't indented like the others.

File: projects/angular/src/data/datagrid/datagrid-column.ts [hunk 2]
```diff
@@ -186,6 +186,7 @@ export class ClrDatagridColumn<T = any>
       this._field = field;
 
       if (!this._sortBy) {
+        console.log('sort by ', field);
```
Comment (kevinbuhmann) [inline projects/angular/src/data/datagrid/datagrid-column.ts:189]: This should be removed.

File: .storybook/stories/datagrid/datagrid-placeholder.stories.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+import { ClrConditionalModule, ClrDatagridModule, ClrDatagridPlaceholder } from '@clr/angular';
+import { Parameters } from '@storybook/addons';
+import { Story } from '@storybook/angular';
+
+import { elements } from '../../helpers/elements.data';
+import { setupStorybook } from '../../helpers/setup-storybook.helpers';
+
+const defaultStory: Story = args => ({
+  template: `
+  <style>.highlight { border: 1px solid red !important; } .electronegativity-container { border-bottom: 4px solid #119cd4; }</style>
```
Comment (kevinbuhmann) [inline .storybook/stories/datagrid/datagrid-placeholder.stories.ts:16]: nitpick: This template isn't indented like the others.

File: .storybook/stories/datagrid/datagrid-placeholder.stories.ts [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  <clr-datagrid
+    ${args.height ? '[style.height.px]="height"' : ''}
+    [ngClass]="{ 'datagrid-compact': compact }">
```
Comment (kevinbuhmann) [inline .storybook/stories/datagrid/datagrid-placeholder.stories.ts:19]: This `>` should be on the next line.

File: .storybook/stories/datagrid/datagrid-placeholder.stories.ts [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+  component: ClrDatagridPlaceholder,
+  argTypes: {
+    // inputs
+    // outputs
+    // methods
+    // story helpers
+    elements: { control: { disable: true }, table: { disable: true } },
+  },
+  args: {
+    // outputs
```
Comment (kevinbuhmann) [inline .storybook/stories/datagrid/datagrid-placeholder.stories.ts:57]: This comment can be removed.

**Review by kevinbuhmann — CHANGES_REQUESTED** [review]
minor comments

File: .storybook/stories/datagrid/datagrid.stories.ts (hunk 1 above)
Comment (steve-haar) [reply .storybook/stories/datagrid/datagrid.stories.ts]: fixed

File: projects/angular/src/data/datagrid/datagrid-column.ts (hunk 2 above)
Comment (steve-haar) [reply projects/angular/src/data/datagrid/datagrid-column.ts]: fixed

File: .storybook/stories/datagrid/datagrid-placeholder.stories.ts (hunk 3 above)
Comment (steve-haar) [reply .storybook/stories/datagrid/datagrid-placeholder.stories.ts]: fixed

File: .storybook/stories/datagrid/datagrid-placeholder.stories.ts (hunk 4 above)
Comment (steve-haar) [reply .storybook/stories/datagrid/datagrid-placeholder.stories.ts]: fixed

File: .storybook/stories/datagrid/datagrid-placeholder.stories.ts [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  props: { ...args },
+});
+
+const defaultParameters: Parameters = {
+  title: 'Datagrid/Placeholder',
+  component: ClrDatagridPlaceholder,
+  argTypes: {
+    // inputs
+    // outputs
+    // methods
```
[the remark this answers is not in the document]
Comment (steve-haar) [reply .storybook/stories/datagrid/datagrid-placeholder.stories.ts]: fixed

File: .storybook/stories/datagrid/datagrid-placeholder.stories.ts (hunk 5 above)
Comment (steve-haar) [reply .storybook/stories/datagrid/datagrid-placeholder.stories.ts]: fixed

**Review by kevinbuhmann — APPROVED** [review]

**Pull request merged**